.travis.yml

```diff
@@ -10,6 +10,7 @@ branches:
   only:
     - master
     - develop
+    - jazzy_travis
```

Comment:
I am thinking we don't need this line. Can we remove it? Also, you probably may want to rename your branch to `issue.jazzy_travis`.